### ISSUE TITLE
Link to privacy policy and imprint on Metro 2.0 sidebar

### ISSUE THREAD

**Issue by zeitrechnen** [pixelfed/pixelfed#261]
Hi, at least in Germany it is necessary that the user can access the imprint and privacy policy on every page. But with the infinite scrolling mechanism this is not possible any longer.

And it must not be hidden, for example in a drop-down menu.

This is of course an important issue, especially in connection with GDPR.

**Comment by dansup** [comment]
@zeitrechnen I think having the links on the right side of the timeline is the best approach, like IG. Thoughts?

**Comment by trwnh** [comment]
Could also be done with a sticky footer? But putting links on the side implies that something else should be on the side... at least in desktop view.

And one other thing to ask is if this legal limitation counts on "app" pages and not just "web" pages? And technically, it is already visible... you just have to be fast and do it before more content loads.

**Comment by faissaloo** [comment]
Put it in the drop down menu, anything else would just be a waste of screen real estate.

**Comment by zeitrechnen** [comment]
As mentioned in Germany you must reach the legal pages with one click. That means you cannot put the links into a drop-down menu.

I know that this is ugly, especially on mobile devices it is not easy to solve. But it would be nice if as many people as possible could use Pixelfed without legal problems. That's at least my opinion. We should be better than instagram, which doesn't care about privacy and legal issues at all :-) 

As a matter of fact: The operator of an instance must make both the imprint and the data protection notice available for his service as described.

In addition, those who create a profile must also make their own imprint and data protection information available. This could be done in the profile page, for example, by simply linking to external pages. It is only important that the links are always visible in the profile.

Last week the Court of Justice of the European Union ruled that someone who runs a Facebook page is jointly responsible for data processing on Facebook. Lawyers assume that this adjudication also applies to other social media platforms.

If user tracking takes place in the instance, it may be necessary to obtain the user's consent before tracking. However, there is a great deal of confusion here at the moment.

Details may be created by the new ePrivacy regulation.

I'll see if I can think of a wise proposition.

**Comment by kpcyrd** [comment]
> In addition, those who create a profile must also make their own imprint and data protection information available.

I don't think this is the case, I don't know a single German site that asks its users to publish an imprint. The german imprint law also explicitly mentions commercial use.

**Comment by zeitrechnen** [comment]
@kpcyrd 

This is a misunderstanding. I don't mean the commentators, but everyone who has a profile in Pixelfed, for example. You are absolutely right that purely private use does not require any imprint. But if, for example, an association or a museum or people who contribute to opinion-forming have a profile, then unfortunately they would have to include an imprint.

That is, at least, my understanding.

**Comment by hellcp** [comment]
@zeitrechnen that applies to websites hosted in Germany, look at Heinlein Support website and openSUSE websites, they all legally need to have imprints. Frankly, we still need to make it convenient for Germans to host their instances :D

**Comment by joachimesque** [comment]
Why isn't the infinite scroll available after a click?

When you visit the page, go to the bottom, there's a button with "load more" at the end of the picture list. You can click on any footer link, or the "load more" button, which loads more photos. A button "always load more" could remember that choice for a time.

**Comment by dansup** [comment]
Fixed in latest commits

**Comment by MrAn0nym** [comment]
No legal expert but as far as I can tell Metro 2.0 reintroduces this issue

**Comment by trwnh** [comment]
> No legal expert but as far as I can tell Metro 2.0 reintroduces this issue

yes, although this is not because of infinite scrolling, it's because the links simply present on the sidebar for some reason...

**Comment by michaelschefczyk** [comment]
It would be very kind if anyone could point out a way to add an imprint to a pixelfed installation.

Under support, it seems to be impossible to generate a response: https://github.com/pixelfed/support/issues/146

It can be done, as this example does show: https://pixelfed.de/site/legal

So far, I was unable to reach the person operating pixelfed.de requesting advice on how to add such imprint.

**Comment by escuco** [comment]
Since one of the last updates it's possible to generate new pages like legal-notice (Impressum in German).

Example from my instance: [https://pixelfed.cafe/site/legal-notice](https://pixelfed.cafe/site/legal-notice)

But unfortunately the link doesn't show up anywhere, e.g. in the footer: 
![screenshot-footer](https://user-images.githubusercontent.com/1647405/189616448-fb5a9f68-55ad-4b65-9695-1c142a475558.jpg)

Is there any way to implement a page link to the footer or under the left menu bar?

Maybe this could resolve the issue with the legal notice / imprint, which is mandatory in Germany.

**Comment by shleeable** [comment]
This issue should be resolved with [#5606](https://github.com/pixelfed/pixelfed/pull/5606).
